src/web/parserUtils.ts

```diff
@@ -179,6 +185,66 @@ function moveCursor(isFocused: boolean, alwaysMoveCursorToTheEnd: boolean, curso
   }
 }
 
+function hidePreBlockBackgrounds(target: HTMLElement) {
+  const preBlocks = [...target.querySelectorAll('*[data-type="pre"]')];
+  const preBlockBackgrounds = [...target.querySelectorAll('.pre-block-background')];
+  for (let i = preBlocks.length - 1; i < preBlockBackgrounds.length - 1; i++) {
+    preBlockBackgrounds[i]?.remove();
+  }
+}
+
+function handlePreBlockBackground(target: HTMLElement, markdownStyle: PartialMarkdownStyle = {}) {
+  hidePreBlockBackgrounds(target);
+  const preBlocks = [...target.querySelectorAll('*[data-type="pre"]')];
+  const preBlockBackgrounds = [...target.querySelectorAll('.pre-block-background')];
```

Comment:
We have the same code in `hidePreBlockBackgrounds` function, can't we query them one time and use it in both places?